nginx/Dockerfile

```diff
@@ -21,39 +21,46 @@ RUN --mount=type=cache,id=download-downloads-${TARGETARCH},sharing=locked,target
     chmod a+x /usr/bin/composer && \
     cleanup.sh
 
+ENV \
+    # renovate: datasource=repology depName=alpine_3_20/icu-data-full
+    ICU_VERSION=74.2-r0 \
+    # renovate: datasource=repology depName=alpine_3_20/nginx
+    NGINX_VERSION=1.26.2-r0	\
+    # renovate: datasource=repology depName=alpine_3_20/php83
+    PHP_VERSION=8.3.15-r0
+
 # Platform specific does require arch specific identifier.
 RUN --mount=type=cache,id=nginx-apk-${TARGETARCH},sharing=locked,target=/var/cache/apk \
     apk add \
-        icu-data-full \
-        nginx \
-        php83 \
-        php83-ctype \
-        php83-curl \
-        php83-dom \
-        php83-fileinfo \
-        php83-fpm \
-        php83-gd \
-        php83-iconv \
-        php83-intl \
-        php83-json \
```

Comment:
`php83-json` doesn't appear to be a package

Reply:
Used to be its own package now it resolves to `php83-common` https://pkgs.alpinelinux.org/package/v3.20/community/x86_64/php83-common

Reply:
Safe to ignore cause `php83-iconv`, etc depend on `php83-common`